src/babylon.scene.ts

```diff
@@ -1294,9 +1295,11 @@
         /**
          * Use this method to simulate a pointer down on a mesh
          * The pickResult parameter can be obtained from a scene.pick or scene.pickWithRay
+         * @param pickResult pickingInfo of the object wished to simulate pointer event on
+         * @param pointerEventInit pointer event state to be used when simulating the pointer event (eg. pointer id for multitouch)
          */
-        public simulatePointerDown(pickResult: PickingInfo): Scene {
-            let evt = new PointerEvent("pointerdown");
+        public simulatePointerDown(pickResult: PickingInfo, pointerEventInit: PointerEventInit | undefined): Scene {

```

Comment:
What is the difference between `pointerEventInit: PointerEventInit | undefined` and pointerEventInit?: PointerEventInit`

Reply:
Looks to work the same (no idea why I wrote it that way) so I switched it to ? to be consistent.